pkg/controllers/kepler.go

```diff
@@ -425,9 +434,27 @@ func exporterReconcilers(k *v1alpha1.Kepler, cluster k8s.Cluster) []reconciler.R
 		updater := newUpdaterForKepler(k)
 		rs = append(rs, updater(exporter.NewSCC(components.Full, k)))
 	}
+
 	return rs
 }
 
+func modelServerReconcilers(k *v1alpha1.Kepler, cluster k8s.Cluster) ([]reconciler.Reconciler, error) {
+	ms := k.Spec.ModelServer
+	cm := modelserver.NewConfigMap(components.Full, ms)
+	deploy := modelserver.NewDeployment(ms)
+	svc := modelserver.NewService(ms)
+	pvc := modelserver.NewPVC()
+
+	rs := updatersForResources(k,
+		// namespace scoped
+		cm,
+		deploy,
+		svc,
+		pvc,
+	)
+	return rs, nil
```

Comment:
We need to ensure that the `status` of `kepler` is properly updated based on the state of deployments.